modules/psmisc/23.7/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://gitlab.com/psmisc/psmisc/-/archive/v23.7/psmisc-v23.7.tar.gz",
+    "integrity": "sha256-jyUmznrG70l2RUzWMJX6EORn73Rc8z3E+R3wvXsQuQU=",
+    "strip_prefix": "psmisc-v23.7",
+    "overlay": {
+        "MODULE.bazel": "sha256-pyxtDrrDiFbiXmfvZS6fNnsGtNEBvxUCCGNhTf+GnCM=",
+        "BUILD.bazel": "sha256-BCHXRpf5PbfBOEBdkY3LsS7xvjwV8x7iPtkxEwDZqYk=",
+        "defs.bzl": "sha256-0dQ2N5ZD95eYEJfEFWPiqz97HDww6P/iTWEjsGo4yWo="
```

Comment:
ah, I just realized that the overlay files are actually placed in the same directory as the registry files (presubmit.yml, source.json, etc). I had intended the overlay files to have to be placed in a subdirectory (such as `overlay/`, just like how patches are in `patches/`), but missed this during code review (https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/bzlmod/IndexRegistry.java;drc=96b90baf08172bb55844c2366111a0c39b6f9e14;l=454).

Changing this would technically be backwards-incompatible, but given this was just added in 7.2.0 and isn't really used anywhere yet (or even documented), I think it's permissible to adjust the behavior now (7.2.1 is pending launch). Would you mind submitting a Bazel PR to add the `overlay/` infix (or waiting for such a PR), waiting for 7.2.1 to release, and then updating this PR to use the new overlay structure?

cc @meteorcloudy @fmeum

Reply:
I forgot to add some rationale for why it's bad the overlay files are in the same directory as registry files -- it introduces potential conflicts between registry files and overlay files (for example, you can't add a `source.json` file as an overlay if you wanted to).

Reply:
I sent https://github.com/bazelbuild/bazel/pull/22811.